tools/tiled_ensemble/ensemble_tiler.py

```diff
@@ -0,0 +1,112 @@
+"""Tiler used with ensemble of models."""
```

Comment:
Do we need this here now that is moved to `src/anomalib`?

Reply:
No, I'll delete this.